### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by kyle-fox**

### CODE REVIEW

File: src/epub/text/part-2.xhtml [hunk 1]
```diff
@@ -134,7 +134,7 @@
...
+				<p>The merchant passed to duties of writing important letters and contracts on to him and got into the habit of discussing all important affairs with him. He soon saw that Siddhartha knew little about rice and wool, shipping and trade, but that he acted in a fortunate manner, and that Siddhartha surpassed him, the merchant, in calmness and equanimity, and in the art of listening and deeply understanding previously unknown people. “This Brahmin,” he said to a friend, “is no proper merchant and will never be one, there is never any passion in his soul when he conducts our business. But he has that mysterious quality of those people to whom success comes all by itself, whether this may be a good star of his birth, magic, or something he has learned among Samanas. He always seems to be merely playing with our business affairs, they never fully become a part of him, they never rule over him, he is never afraid of failure, he is never upset by a loss.”</p>
```
Comment (kyle-fox) [inline src/epub/text/part-2.xhtml]: Correction is `our business affairs`.

File: src/epub/text/part-2.xhtml [hunk 2]
```diff
@@ -323,7 +323,7 @@
...
-				<p>“Get the brushwood for yourself!” he shouted foaming at the mouth. “I’m not your servant. I do know, that you won’t hit me, you don’t dare; I do know, that you constantly want to punish me and put me down with your religious devotion and your indulgence. You want me to become like you, just as devout, just as soft, just as wise! But I, listen up, just to make you suffer, I’d rather become a highway-robber and murderer, and go to hell, than to become like you! I hate you, you’re not my father, and if you’ve ten times been my mother’s fornicator!”</p>
+				<p>“Get the brushwood for yourself!” he shouted foaming at the mouth. “I’m not your servant. I do know, that you won’t hit me, you don’t dare; I do know, that you constantly want to punish me and put me down with your religious devotion and your indulgence. You want me to become like you, just as devout, just as soft, just as wise! But I, listen up, just to make you suffer, I’d rather become a highway-robber and murderer, and go to hell, than to become like you! I hate you, you’re not my father, even if you’ve ten times been my mother’s fornicator!”</p>
```
Comment (kyle-fox) [inline src/epub/text/part-2.xhtml]: Correction is `even if you've ten times been`

File: src/epub/text/part-2.xhtml [hunk 3]
```diff
@@ -350,7 +350,7 @@
...
...d over, came back at him from his counterpart. To show his wound to this listener was the same as bathing it in the river, until it had cooled and become one with the river. While he was still speaking, still admitting and confessing, Siddhartha felt more and more that this was no longer Vasudeva, no longer a human being, who was listening to him, that this motionless listener was absorbing his confession into himself like a tree the rain, that this motionless man was the river itself, that he was God himself, that he was the eternal itself. And while Siddhartha stopped thinking of himself and his wound, this realisation of Vasudeva’s changed character took possession of him, and the more he felt it and entered into it, the less wondrous it became, the more he realised that everything was in order and natural, that Vasudeva had already been like this for a long time, almost forever, that only he had not quite recognised it, yes, that he himself had almost reached the same state. He felt, that he was now seeing old Vasudeva as the people see the gods, and that this could not last; in his heart, he started bidding his farewell to Vasudeva. Through all this, he talked incessantly.</p>
```
Comment (kyle-fox) [inline src/epub/text/part-2.xhtml]: Correction is `Through all this`

**Comment by acabal** [comment]
Thanks!

This translation is very poor so I'm glad to have as many typo fixes and editorial corrections as possible.

**Comment by kyle-fox** [comment]
No problem - thanks for working on this!

**Pull request merged**